kubernetes/keytransparency-deployment.yml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: kt-server
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        run: kt-server
+    spec:
+      containers:
+      - name: kt-server
+        image: us.gcr.io/transparent-keyserver/keytransparency-server
```

Comment:
I've created a new service-key in this project but authenticating using this key does not grant the permissions to create a repository for the container registry there. Either one of the owners (@gdbevlin, @cesarghali)  need to create a key-file, download it and encrypt it after we merged this pull or you need to grant me access to do so.